cmd/internal/flag.go

```diff
@@ -102,6 +106,12 @@ func initRegistryClientFlags() {
 	flag.StringVar(&registryCredentialHelpers, "registryCredentialHelpers", "", "Credential helpers to enable (default,google,amazon,azure,github). No helpers are added when this flag is empty.")
 }
 
+func initImageVerifyCacheFlags() {
+	flag.BoolVar(&imageVerifyCacheEnabled, "imageVerifyCacheEnabled", true, "Whether to use a TTL cache for storing verified images.")
+	flag.Int64Var(&imageVerifyCacheMaxSize, "imageVerifyCacheMaxSize", 0, "Max size limit for the TTL cache, 0 means deafult 1000 size limit.")
```

Comment:
@vishal-chdhry @eddycharly Is it okay?? I have added checks in cache for default values